netops_demo/py/functions/predict/predict.py

```diff
@@ -0,0 +1,162 @@
+import os
+import json
+import pickle
+import http.client
+import sklearn
+import numpy as np
+
+import libs.utils.utils
+import libs.nuclio_sdk
+
+
+def predict(context, event):
+    if event.path == '/configure':
+        if type(event.body) is not dict:
+            return context.Response(status_code=400)
+
+        _configure(context, event.body)
+    elif event.path == '/start':
+        _start(context)
+    elif event.path == '/stop':
+        _stop(context)
+    elif event.path in ['/predict', '/', '']:
+        if context.user_data.state == 'predicting':
+            return _predict_error(context, **(event.body or {}))
+    else:
+        context.logger.warn_with('Got unsupported path', method=event.method, path=event.path)
+        return context.Response(status_code=400)
+
+
+def init_context(context):
+    # Init states
+    libs.utils.utils.init_context(context)
+
+    # check to see if there's an environment variable with our initial configuration
+    if 'PREDICTOR_CONFIGURATION' in os.environ:
+        _configure(context, json.loads(os.environ['PREDICTOR_CONFIGURATION']))
+
+
+def _configure(context, configuration: dict):
+    # Load model from given path
+    try:
+        if 'MODEL_FILE' in os.environ:
+            with open(os.environ.get('MODEL_FILE'), 'rb+') as f:
+                context.user_data.model = pickle.load(f)
+    except IOError:
+        context.user_data.model = None
```

Comment:
Do we really want to silently ignore the error here? At the very least add a warning log